app/stories/pages/Home.stories.tsx

```diff
@@ -1,6 +1,6 @@
-import { ComponentStory, ComponentMeta } from '@storybook/react'
+import { ComponentMeta, ComponentStory } from '@storybook/react'
```

Comment:
This file should be deleted. I might have accidentally re-introduced it in the merge conflict resolution.